frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -457,3 +457,37 @@ export type Status = {
   reason: string;
   status: string;
 };
+
+export enum BadgeType {
+  DEV = 'Dev Preview',
+  TECH = 'Tech Preview',
+}
+
+export type K8sKind = {
```

Comment:
@invincibleJai I think there was a reason this was not brought over in one of your previous PRs, right? Sam is off for the week and I don't recall the reasoning (that I'm pretty sure he made).